CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file. This
 change log follows the conventions of
 [keepachangelog.com](http://keepachangelog.com/).
 
+## 3.8.4 / 2023-02-08
+- deprecate the matcher-combinators.midje ns and functions in it
+- move midje dependency to :scope "provided"
+  - this eliminates the transitive dep for those not using midje features
```

Comment:
These two lines are no longer valid. #197 (since merged into this PR) removed `project.clj` and effectively already removed midje as a transitive dependency.